GITHUB_ORG_MANGEMENT_POLICY.md

```diff
@@ -1,10 +1,5 @@
 # Node.js GitHub Organization Management Policy
 
-The Node.js Foundation GitHub Organization (https://github.com/nodejs) is
```

Comment:
Would there be a counterpart in the new version of this doc that described the GitHub Org as a development resource? Unless this has been superseded in another doc.

Reply:
@keywordnew added back some of the text.  Was not clear to me that the Foundation provides the resource so wanted to remove that, but the reference to the org is good to retain.